spec/support/simple_server.rb

```diff
@@ -155,7 +155,7 @@ def do_GET(request, response) # rubocop:disable Naming/MethodName
       if @simple_server.routes.include?(request.path)
         @simple_server.routes[request.path].call request, response
       else
-        STDERR.puts "[SimpleServer] no route found: #{request.path}"
```

Comment:
New Rubocop version complained about this?

Reply:
Yup - “please use warn” - so that way users can squelch the output if they want to.